vectorsearch/test_procedures/default.json

```diff
@@ -56,7 +56,8 @@
                 "data_set_path" : "{{ query_data_set_path | default('/tmp/vector-dataset.hdf5') }}",
                 "num_vectors" : {{ query_count | default(-1) }},
                 "id-field-name": "{{ id_field_name }}"
-            }
+            },
+            "clients": {{ query_clients | default(1)}}
```

Comment:
Since this is a part of the standard OSB workloads, we should remain consistent with other workloads and use `search_clients` instead of `query_clients`.

Reply:
Updated